Makefile

```diff
@@ -1,4 +1,10 @@
-prefix ?= /usr/
+UNAME:= $(shell uname -s)
+
+ifeq ($(UNAME), Darwin)
+	prefix ?= /usr/local
+else
+	prefix ?= /usr
+endif
 
 all: podget podget.7.gz changelog.gz
 
```

Comment:
What branch are you basing this code off?   I've already added these changes to the dev-machomebrew branch.   It would make sense to confine our efforts to that branch so they can all be merged at once when we are happy with them.